Std/Data/Nat/Lemmas.lean

```diff
@@ -356,10 +356,10 @@ protected theorem eq_zero_of_add_eq_zero_right (h : n + m = 0) : n = 0 :=
 protected theorem eq_zero_of_add_eq_zero_left (h : n + m = 0) : m = 0 :=
   (Nat.eq_zero_of_add_eq_zero h).2
 
-protected theorem add_eq_zero {n m : Nat} : n + m = 0 ↔ n = 0 ∧ m = 0 :=
-  ⟨eq_zero_of_add_eq_zero, by simp (config := {contextual := true})⟩
+protected theorem add_eq_zero_iff : n + m = 0 ↔ n = 0 ∧ m = 0 :=
```

Comment:
Not sure the name change is necessary. What else would `Nat.add_eq_zero` mean?

Reply:
I don't know what diff you're commenting on here, but this name is in main already:

https://github.com/leanprover/std4/blob/fa17cba0459830906e99f0a81fb0edc106220380/Std/Data/Nat/Lemmas.lean#L358-L359